### PR TITLE
SAK-31367 Fix SQL issues in migration scripts.

### PR DESCRIPTION
**Pull request by buckett**
Fix up SQL bugs in Sakai 11 migration scripts. The commits are made against the issues that originally made the changes so it's easier to see in a git blame what the change was.

What do people think?

### CODE REVIEW

File: reference/docs/conversion/sakai_11_mysql_conversion.sql [hunk 1]
```diff
@@ -681,27 +681,27 @@ alter table lti_tools add pl_importitem tinyint(4) default 0;
...
+ALTER TABLE lti_deploy MODIFY pagetitle VARCHAR(1024);
+ALTER TABLE lti_deploy ADD allowcontentitem tinyint(4) DEFAULT 0;
+ALTER TABLE lti_deploy MODIFY reg_key VARCHAR(1024);
+ALTER TABLE lti_deploy MODIFY reg_password VARCHAR(1024);
+ALTER TABLE lti_deploy ADD reg_ack TEXT;
+ALTER TABLE lti_deploy MODIFY consumerkey VARCHAR(1024);
+ALTER TABLE lti_deploy MODIFY secret VARCHAR(1024);
+ALTER TABLE lti_deploy ADD new_secret VARCHAR(1024);
 
 CREATE TABLE lti_memberships_jobbbs (
```
Comment (bjones86) [inline reference/docs/conversion/sakai_11_mysql_conversion.sql]: Shouldn't all these lti* tables be upper case? Same with the column names.

Comment (buckett) [inline reference/docs/conversion/sakai_11_mysql_conversion.sql]: On our case sensitive deployment all the table names are lower case and the `org.sakaiproject.lti.impl.DBLTIService#init` looks to create them all as lower case. 

I think MySQL is not case sensitive for columns names (http://dev.mysql.com/doc/refman/5.7/en/identifier-case-sensitivity.html)

Personally I like all the SQL grammar in upper case and schema references in lower case as I find it easier to read, but that's just a personal thing.

Comment (ottenhoff) [inline reference/docs/conversion/sakai_11_mysql_conversion.sql]: Yeah the table was created in lowercase so it needs to be consistently referred to in lowercase.

File: reference/docs/conversion/sakai_11_mysql_conversion.sql [hunk 2]
```diff
@@ -613,8 +613,8 @@ alter table SAM_ITEM_T add column MIN_SCORE double NULL;
 alter table SAM_PUBLISHEDITEM_T add column MIN_SCORE double NULL;
 
 -- KNL-1405 Don't have defaults for TIMESTAMP in SAKAI_SESSION
-ALTER TABLE SAKAI_SESSION MODIFY SESSION_START NULL;
-ALTER TABLE SAKAI_SESSION MODIFY SESSION_END NULL;
+ALTER TABLE SAKAI_SESSION MODIFY SESSION_START TIMESTAMP NULL;
+ALTER TABLE SAKAI_SESSION MODIFY SESSION_END TIMESTAMP NULL;
```
Comment (ottenhoff) [inline reference/docs/conversion/sakai_11_mysql_conversion.sql]: Is this an opinionated change or a change being pulled from somewhere else?

Comment (buckett) [inline reference/docs/conversion/sakai_11_mysql_conversion.sql]: On MySQL 5.5.46 without the column type specification I get an error of:

    Error Code: 1064. You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'NULL' at line 1

**Pull request merged**